### ISSUE TITLE
Update ZAP docker images regularly

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zaproxy#6669]
I asked, and people answered: https://twitter.com/psiinon/status/1410229032105517058

![Screenshot 2021-07-05 at 15-18-04 Simon Bennetts on Twitter](https://user-images.githubusercontent.com/1081115/124477376-4233d400-dda4-11eb-8cb3-261ad0683f80.png)

I'm guessing that just updating the base image is easier - we just have to rerun the [Release main Docker](https://github.com/zaproxy/zaproxy/actions/workflows/release-main-docker.yml) action?
However I suspect updating the add-ons will be a bit more work.
Should we start with just updating the base image and then updating the add-ons?
How often should we update the images - one a month, once a week or ??

**Comment by thc202** [comment]
How are those images going to be tagged? They will no longer be 2.10.0 (or whatever), right?

**Comment by psiinon** [comment]
If they are not tagged as 2.10.0 (or whatever) then anyone using that tag wont use them.
So I think people will _expect_ them to stay with the same tag. Unless this has some side effect thats really undesirable.

**Comment by thc202** [comment]
True, as long as we are clear that the add-ons are updated regularly I guess it's fine.
